frontend/src/components/SamplesList.tsx

```diff
@@ -243,6 +260,20 @@ export default function SamplesList({
 
   return (
     <>
+      <Container fluid>
+        {!parentWhereVariables && <BreadCrumb currPageTitle="samples" />}
+        <Title
+          text={
+            popupParamId
+              ? `Viewing ${parentDataName?.slice(
+                  0,
+                  -1
+                )} ${popupParamId}'s samples`
+              : "samples"
+          }
+        />
+      </Container>
```

Comment:
These are components that render the breadcrumbs and title of the Samples view.

They previously lived in the parent component (either `RecordsList` or `SamplesPage`). I moved them here so that the Patient Samples view's title can dynamically show `cmoPatientId`/`dmpPatientId`. 

Only by being inside `SamplesList` that the title component can look at the fetched data and access `cmoPatientId`/`dmpPatientId` value accordingly.